### ISSUE TITLE
[BUG] Wrong default background color if editing Tags in color picker window

### ISSUE THREAD

**Issue by stho90** [SillyTavern/SillyTavern#1793]
### Environment

Self-Hosted (Bare Metal)

### System

Firefox 122.0

### Version

v1.11.2 (release) + v1.11.3 (release)

### Describe the problem

Hello,

i have notified a small bug. To reproduce it, i did the following:
- I used a fresh new installed SillyTavern v1.11.2
- Then i disabled the automatic tag import in the settings and added some char cards
- When i clicked the "Manage Tags" button in the char overview, i wanted to change some background colors for my tags
- I noted that the current color is "000000"
- I selected a color (like blue) for one tag but wanted to reset it. So i changed it back to "000000"
- The tag now looks different, so i made a short analysis. Have a look at this:

![grafik](https://github.com/SillyTavern/SillyTavern/assets/143583451/cfbb0d77-9c2f-4cae-a082-60b6b932a740)

The issue comes here:
- The tag "Thief" was manually set to "000000"
- The tag "Wife" was created by the char details view without changing the color
- The newly created tag (from the char details view) has the color code "101010", but if i select it, the color picker shows "000000"

![grafik](https://github.com/SillyTavern/SillyTavern/assets/143583451/2aa1e858-6b1b-439e-a55c-0a8cccf0d488)

### Please tick the boxes

- [X] You have explained the issue clearly, and included all relevant info
- [X] You've checked that this [issue hasn't already been raised](https://github.com/SillyTavern/SillyTavern/issues?q=is%3Aissue)
- [X] You've checked the [docs](https://docs.sillytavern.app/) ![important](https://img.shields.io/badge/Important!-F6094E)

**Comment by Cohee1207** [comment]
I don't see any huge issues here.

"000000" here means the lack of color (by default tag colors are unassigned) instead of pitch black. UI control displays null colors as black by default, making any change there will set a selected color.

"101010" is the default/fallback 30% transparent black background evaluated against the background color of the tag manager, giving this value in the mixture.

**Comment by stho90** [comment]
This ticket is still open

**Comment by Wolfsblvt** [comment]
Not really. The whole color selection for tags was reworked, and you now have a button to reset/rebind the tag to the theme color. Which works.
Please test this again.
